File: Programming Basics with Python/02 Conditional statements/More exercises/06_Pets.py
import math

days = int(input())
food_kg = int(input())
dog_food_kg = float(input())
cat_food_kg = float(input())
turtle_food_g = float(input())
needed_food = days * dog_food_kg + days * cat_food_kg + days * turtle_food_g / 1000
result = food_kg - needed_food
if food_kg >= needed_food:
    print(f'{math.floor(result)} kilos of food left.')
else:
    print(f'{math.ceil(abs(result))} more kilos of food are needed.')

File: JS Basics/02 Conditional Statements - More Exercises/06. Pets.js
function pets(days, foodKg, dayDogfoodKg, dayCatFoodKg, dayTurtleFoodG) {
    foodKg = Number(foodKg);
    let neededFood = Number(days) * (Number(dayDogfoodKg) + Number(dayCatFoodKg) + (Number(dayTurtleFoodG) / 1000));
    let result = Math.abs(foodKg - neededFood);
    if (foodKg >= neededFood) {
        console.log(`${Math.floor(result)} kilos of food left.`);
    }
    else {
        console.log(`${Math.ceil(result)} more kilos of food are needed.`);
    }
}
pets(5, 10, 2.1, 0.8, 321)